HeterogeneousCore/AlpakaCore/src/module_backend_config.cc

```diff
@@ -17,7 +17,7 @@ namespace cms::alpakatools {
     descAlpaka.addUntracked<std::string>("backend", "")
         ->setComment(
             "Alpaka backend for this module. Can be empty string (for the global default), 'serial_sync', or "
-            "'cuda_async'");
+            " - depending on the architecture and available hardware - 'cuda_async', 'rocm_async'");
```

Comment:
I think not for the comment...

Reply:
In fact, at the moment this is just not possible: _in CMSSW_ we do not declare all "available" alpaka backends; we declare only a single `ALPAKA_..._ENABLED` macro at a time, when building _that_ backend.

Unlike in `pixeltrack-standalone`, so far we have managed to get away with it (and without the need of forward declarations, _etc._) thanks to the plugin-based nature of CMSSW (and to the changes we pushed to Alpaka itself over the last two years to avoid ODR violations).

Reply:
Good point. This use case is certainly not enough to justify the additional complexity.